api/models/schema.py

```diff
@@ -52,7 +53,8 @@ class Person(SchemaBaseModel):
 class Organization(SchemaBaseModel):
     type: str = Field(
         alias="@type",
-        default="Organization"
+        default="Organization",
+        const=True
```

Comment:
The `const` property is not needed here because we are adding a default and making the field hidden in the UI schema, so the value is always loaded.

Reply:
I discovered this during unit tests. We have the creator set to be union type of Person | Organization (https://github.com/I-GUIDE/catalogapi/blob/33-fix-tests/api/models/schema.py#L382).

Without making the type constant for Person and Organization, when I was passing data for a person as the creator, pydantic was creating an Organization instance as the creator and vice-versa.

Reply:
const has been removed from pydantic v2. https://docs.pydantic.dev/latest/migration/#changes-to-pydanticfield

Reply:
I think you could probably use frozen to be forward compatible with v2

Reply:
Seems frozen is available in pydantic v2.  Will create a separate issue to upgrade to v2.